index.html

```diff
@@ -1,20 +1,291 @@
+						<g id="facebook">
+							<path id="icon"
+								d="M9.04598 5.865V8.613H7.03198V11.973H9.04598V21.959H13.18V11.974H15.955C15.955 11.974 16.215 10.363 16.341 8.601H13.197V6.303C13.197 5.96 13.647 5.498 14.093 5.498H16.347V2H13.283C8.94298 2 9.04598 5.363 9.04598 5.865V5.865Z"
+								fill="white" />
+						</g>
+					</svg>
+				</a>
+			</div>
+		</nav>
+		<!-- navigation menu  -->
+		<nav class="navbar relative bg-transparent md:bg-white md:shadow-sm">
+			<div class="navbar-inner mx-auto flex items-center justify-between gap-4 py-3 md:w-10/12">
+				<button id="hamburger-button" class="hamburger flex-start bg h-10 cursor-pointer md:hidden">
+					<i class="fa-sharp fa-solid fa-bars" style="font-size: 34px"></i>
+				</button>
+
+				<!-- logo  -->
+				<div>
+					<a href="index.html" class="logo items-center gap-2 leading-6 transition-all md:flex">
+						<img src="./public/images/logo-arabic2.jpg" style="width: 144px" alt="logo" />
+					</a>
+				</div>
+
+				<div class="relative flex items-center justify-center">
+					<ul class="hidden items-center gap-9 gap-y-4 text-xl md:flex md:flex-wrap" aria-label="main">
+						<li class="text-coral-red hover:text-coral-red hover:opacity-90">
+							<a href="./about.html" class=""> About </a>
+						</li>
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#program" class=""> Program </a>
+						</li>
+
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#join" class=""> Join </a>
+						</li>
+
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#partner" class=""> Sponsor </a>
+						</li>
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#" class=""> News </a>
+						</li>
+						<li
+							class="border-2 border-coral-red py-2 px-4 transition-all hover:bg-coral-red hover:text-white hover:opacity-90">
+							<a href="index.html" class=""> CC Campaign </a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+
+		<!-- Mobile menu -->
+		<nav id="mobile-menu"
+			class="mobile-menu absolute top-0 hidden w-full origin-top animate-times-menu-animation flex-col justify-center bg-metal px-4 py-3 text-4xl text-white">
+			<button class="close self-start">X</button>
+			<div class="flex h-screen-94h flex-col items-center gap-4 py-8" aria-label="mobile-menu">
+				<a href="index.html" class="w-full nav-link px-4 text-center hover:opacity-70">Home</a>
+
+				<a href="./about.html" class="w-full nav-link px-4 text-center hover:opacity-70">About</a>
+				<a href="#program" class="nav-link w-full px-4 text-center hover:opacity-70">Program</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Join</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Sponsor</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">News</a>
+			</div>
+		</nav>
```

Comment:
- Just like the previous reviewer mentioned, please make sure that all your navigation links are wrapped into just one `nav` tag. It would help to improve the semantics of your site. Please fix this for both the home and about page 😄